router/transformer/transformer_test.go

```diff
@@ -813,8 +813,8 @@ var oauthv2ProxyTestCases = []oauthv2ProxyTcs{
 			ProxyRequestResponseBody: `{"message": "some message that we got from transformer","authErrorCategory":"REFRESH_TOKEN","response":[{"statusCode":401,"error":"token has expired","metadata":{"workspaceId":"workspace_id","destinationId":"destination_id","jobId":2}},{"statusCode":401,"error":"token has expired","metadata":{"workspaceId":"workspace_id","destinationId":"destination_id","jobId":1}}]}`,
 			ProxyRequestStatusCode:   500,
 			RespStatusCodes: map[int64]int{
-				1: 401,
-				2: 401,
+				1: 500,
+				2: 500,
```

Comment:
Similar for below as well.
```suggestion
				1: http.StatusInternalServerError,
				2: http.StatusInternalServerError,
```

Reply:
Done